### ISSUE TITLE
How do I change the face normals after instantiating a Mesh object?

### ISSUE THREAD

**Issue by PythoneerSamurai** [facebookresearch/pytorch3d#1951]
Greetings.
Could you let me know how I can change a Mesh's face normals after it has been instantiated?
Also, I want to know if changing a face's normal will change it's orientation or not.
I'm working on mesh smoothing and want to see if I can get some good results by changing the face normals, since the common loss functions of pytorch3d are not smoothing my cube mesh.

**Comment by bottler** [comment]
If you are doing gradient based optimization, it is recommended to instantiate a new Meshes object in each iteration, so you don't need to "change" the normals in an existing mesh. Changing them is possible if you look at the implementation, but if you are using gradients you need to be careful about how autograd works.
